src/unix/print.js

```diff
@@ -8,6 +8,18 @@ const getCommand = (pdf, options) => {
   if (options.printer) {
     command += " -d " + options.printer;
   }
+  // check for other options
+  Object.keys(options).forEach(function(key) {
+    if (key !== "printer") {
+      const value = options[key];
+      if (value !== false) {
```

Comment:
why not just `if (!!value) {`

Reply:
because f.e. `{ 'some-option': 0 }` would not be serialized properly with `!! value`